### PR TITLE
New MODIS and STATSGO veg/soil data.

### LINKED ISSUE #702: Incorporate global 30-sec MODIS vegetation data and STATSGO soil data.
Issue #698 was open to add the 30-sec VIIRS vegetation data. The MODIS vegetation data and STATSGO soil data are also available to 30-sec. Currently, only coarser resolution versions are available. Update UFS_UTILS to use the 30-sec data.

### PR DESCRIPTION
**Pull request by GeorgeGayno-NOAA**
## DESCRIPTION OF CHANGES: 
Updates to use the MODIS vegetation data and STATSGO soil data at its full 30-sec resolution. Currently, only coarser resolution versions are available. 

## TESTS CONDUCTED: 
- The grid_gen consistency tests were run on all supported machines and passed as expected.
- A test regional CONUS grid was created (1) using the 30 second data and (2) the 0.05-degree data. The model grid soil and vegetation type compared favorably between the two tests.
- @BenjaminBlake-NOAA did his own independent tests.

For more test details, see #702.

## DEPENDENCIES:
The new data files were pushed to the 'fix' directories on all machines by @KateFriedman-NOAA.

## DOCUMENTATION:
New data sources were added to readthedocs.:
https://georgegayno-noaaufs-utils.readthedocs.io/en/feature-vegsoil30s/ufs_utils.html#id57

## ISSUE: 
Fixes #702

### CODE REVIEW

**Review by BenjaminBlake-NOAA — APPROVED** [review]
The changes look good to me.

**Comment by GeorgeGayno-NOAA** [comment]
@JeffBeck-NOAA or @LarissaReames-NOAA, please review when you get a chance.

File: driver_scripts/driver_grid.jet.sh [hunk 1]
```diff
@@ -53,10 +53,11 @@
...
-#   9) Check settings for 'make_gsl_orog' and 'veg_type_src'
-#      below.
-#  10) Submit script: "sbatch $script".
-#  11) All files will be placed in "out_dir".
+#   9) To use GFS orographic drag suite, set 'make_gsl_orog' to true.
+#  10) Set 'soil_veg_src' and 'veg_type_src' to choose the
+#      soil type and vegetation type data.
+#  11) Submit script: "sbatch $script".
+#  12) All files will be placed in "out_dir".
 #
```
Comment (JeffBeck-NOAA) [inline driver_scripts/driver_grid.jet.sh:61]: Should this be "To use the GSL orographic drag suite"?

File: driver_scripts/driver_grid.orion.sh [hunk 2]
```diff
@@ -52,10 +52,11 @@
...
-#      below.
-#  10) Submit script: "sbatch $script".
-#  11) All files will be placed in "out_dir".
+#   9) To use GFS orographic drag suite, set 'make_gsl_orog' to true.
+#  10) Set 'soil_veg_src' and 'veg_type_src' to choose the
+#      soil type and vegetation type data.
+#  11) Submit script: "sbatch $script".
+#  12) All files will be placed in "out_dir".
 #
 #-----------------------------------------------------------------------
```
Comment (JeffBeck-NOAA) [inline driver_scripts/driver_grid.orion.sh:61]: Same comment as above.

File: driver_scripts/driver_grid.wcoss2.sh [hunk 3]
```diff
@@ -51,10 +51,11 @@
...
-#   9) Check settings for 'make_gsl_orog' and 'veg_type_src'
-#      below.
-#  10) Submit script: "cat $script | bsub".
-#  11) All files will be placed in "out_dir".
+#   9) To use GFS orographic drag suite, set 'make_gsl_orog' to true.
+#  10) Set 'soil_veg_src' and 'veg_type_src' to choose the
+#      soil type and vegetation type data.
+#  11) Submit script: "qsub $script".
+#  12) All files will be placed in "out_dir".
 #
```
Comment (JeffBeck-NOAA) [inline driver_scripts/driver_grid.wcoss2.sh:59]: Again, similar comment.

**Review by JeffBeck-NOAA — COMMENTED** [review]
Thanks for getting this implemented!  It will definitely help the SRW App and RRFS.  I just had a question about the GSL drag suite comment in a few files.

File: driver_scripts/driver_grid.jet.sh (hunk 1 above)
Comment (GeorgeGayno-NOAA) [reply driver_scripts/driver_grid.jet.sh:61]: Yes. Made the update at 65e1205.

File: driver_scripts/driver_grid.orion.sh (hunk 2 above)
Comment (GeorgeGayno-NOAA) [reply driver_scripts/driver_grid.orion.sh:61]: Done at 65e1205.

File: driver_scripts/driver_grid.wcoss2.sh (hunk 3 above)
Comment (GeorgeGayno-NOAA) [reply driver_scripts/driver_grid.wcoss2.sh:59]: Done at 65e1205.

**Comment by GeorgeGayno-NOAA** [comment]
> Thanks for getting this implemented! It will definitely help the SRW App and RRFS. I just had a question about the GSL drag suite comment in a few files.

@JeffBeck-NOAA Did I address your concerns?

**Review by JeffBeck-NOAA — APPROVED** [review]
Looks good!  Thanks, @GeorgeGayno-NOAA

**Pull request merged**